### PR TITLE
Remove non- stdlib errors from `lxd/response`

### PR DESCRIPTION
**Pull request by markylaing**
The package is being consumed by microcluster and microcloud. To avoid import conflicts with other versions of DQLite, we need to remove it's import.

### CODE REVIEW

**Review by tomponline — APPROVED** [review]
Thanks.

@roosterfish please let me know if this is OK to backport to stable-5.21?

File: lxd/db/errors.go [hunk 1]
```diff
@@ -2,9 +2,19 @@ package db
...
+	"github.com/mattn/go-sqlite3"
 )
 
 var (
 	// ErrNoClusterMember is used to indicate no cluster member has been found for a resource.
 	ErrNoClusterMember = fmt.Errorf("No cluster member found")
 )
+
+// SmartErrors are used to return more appropriate errors to the caller.
+var SmartErrors = map[int][]error{
```
Comment (roosterfish) [inline lxd/db/errors.go:17]: So in case of `microcluster/v2` we create our own `SmartErrors` using `go-dqlite/v2`?

Comment (roosterfish) [reply lxd/db/errors.go:17]: And the call `Init()` using our own to prevent importing the `v3`.

**Review by roosterfish — APPROVED** [review]

File: lxd/db/errors.go (hunk 1 above)
Comment (tomponline) [reply lxd/db/errors.go:17]: yeah thats right

**Pull request merged**